tensorflow/core/platform/default/logging.cc

```diff
@@ -109,14 +109,29 @@ int64 MinLogLevel() {
   }
 }
 
+int64 MinLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_LOG_LEVEL");
+  return LogLevelStrToInt(tf_env_var_val);
+}
+
+int64 MinVLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_VLOG_LEVEL");
```

Comment:
Use std::istringstream should work, Updated patch to use istringstream and also addressed your above comment: https://github.com/tensorflow/tensorflow/pull/6375#discussion-diff-93358241R43.

Please let me know your thoughts, thanks!